applications/RANSApplication/custom_processes/rans_nut_nodal_update_process.cpp

```diff
@@ -0,0 +1,134 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// System includes
+#include <cmath>
+#include <limits>
+
+// External includes
+
+// Project includes
+#include "includes/cfd_variables.h"
+#include "includes/define.h"
+#include "utilities/parallel_utilities.h"
+
+// Application includes
+#include "rans_application_variables.h"
+
+// Include base h
+#include "rans_nut_nodal_update_process.h"
+
+namespace Kratos
+{
+RansNutNodalUpdateProcess::RansNutNodalUpdateProcess(
+    Model& rModel,
+    Parameters rParameters)
+: mrModel(rModel)
+{
+    KRATOS_TRY
+
+    rParameters.ValidateAndAssignDefaults(GetDefaultParameters());
+
+    mEchoLevel = rParameters["echo_level"].GetInt();
+    mModelPartName = rParameters["model_part_name"].GetString();
+
+    KRATOS_CATCH("");
+}
+
+RansNutNodalUpdateProcess::RansNutNodalUpdateProcess(
+    Model& rModel,
+    const std::string& rModelPartName,
+    const int EchoLevel)
+: mrModel(rModel),
+  mModelPartName(rModelPartName),
+  mEchoLevel(EchoLevel)
+{
+}
+
+int RansNutNodalUpdateProcess::Check()
+{
+    KRATOS_TRY
+
+    const auto& r_model_part = mrModel.GetModelPart(mModelPartName);
+
+    KRATOS_ERROR_IF(!r_model_part.HasNodalSolutionStepVariable(VISCOSITY))
+        << "VISCOSITY is not found in nodal solution step variables list of "
+        << mModelPartName << ".";
```

Comment:
Why do you need this if the `VISCOSITY` is retrieved from the properties?

Reply:
It is only there to support old `VMS` and `FractionalStep` formulations. The new `QSVMS` formulation does not need it :)

Reply:
I made this as a seperte process, so once we update `FractionalStep` and remove old `VMS`, i can remove this whole process :)

Reply:
OK! Thanks for clarifying!